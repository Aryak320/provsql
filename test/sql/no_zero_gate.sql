\set ECHO none
\pset format csv
SET search_path to provsql_test;

SELECT * FROM personnel EXCEPT SELECT * FROM personnel;

SELECT * FROM personnel EXCEPT ALL SELECT * FROM personnel;
