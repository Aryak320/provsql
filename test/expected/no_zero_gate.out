\set ECHO none
id,name,position,city,classification,provsql
id,name,position,city,classification,provsql
